Tests/Kernels/ConsoleTest/Kernel.cs

```diff
@@ -32,100 +32,136 @@ protected override void Run()
-                Console.WriteLine(true);
-                Console.WriteLine("The letter 'A'");
-                Console.WriteLine('A');
-                char[] charBuffer = new char[] { 'A', 'B', 'C' };
-                Console.WriteLine("Then ABC");
-                Console.WriteLine(charBuffer);
-                Console.WriteLine("...42.42");
-                Console.WriteLine(42.42);
-                Console.WriteLine("...42.42 (float)");
-                Console.WriteLine(42.42f);
-                Console.WriteLine("...42");
-                Console.WriteLine(42);
-                Console.WriteLine("...42 (long)");
-                Console.WriteLine(42L);
-                Console.ReadKey();
-                object test = "Test";
-                Console.WriteLine("...Test (as object)");
-                Console.WriteLine(test);
-                Console.WriteLine("The next line should be empty (null object)");
-                object s = null;
-                Console.WriteLine(s);
-                Console.WriteLine("...42 (uint)");
-                Console.WriteLine(42U);
-                Console.WriteLine("...42 (ulong)");
-                Console.WriteLine(42UL);
-                Console.WriteLine("...BC");
-                Console.WriteLine(charBuffer, 1, 2);
-
-                Console.WriteLine("Test Format arg0 {0}", "test");
-                Console.WriteLine("Test Format arg0 {0} arg1 {1}", "test", 42);
-                Console.WriteLine("Test Format arg0 {0} arg1 {1} arg2 {2}", "test", 42, 69.69);
-                Console.WriteLine("Test Format arg0 {0} arg1 {1} arg2 {2} arg3 {3}", "test", 42, 69.69, 25000L);
-                /* String.Format does not support x or X and probably neither the rest of "special" formatting */
-                //Console.WriteLine("Test Format (hex) {0:x}", 42);
-
-                Console.WriteLine("Layout switched to DE...");
-                SetKeyboardScanMap(new DE_Standard());
-                Console.WriteLine("Write in germanic now I'll read it with Console.ReadLine()...");
-
-                var str = Console.ReadLine();
-                Console.WriteLine($"You have written: {str}");
-
-                Console.WriteLine("Write in germanic now I'll read it with Console.ReadKey()...");
-                var character = Console.ReadKey();
-                Console.WriteLine($"You have written: {character.KeyChar}");
-
-                Console.WriteLine("Press any key to terminate this test...");
-
-                Console.ReadKey();
+                TestConsoleEncoding();
+
+                TestVGAResolutions();
+
                 TestController.Completed();
             }
             catch (Exception e)
             {
                 mDebugger.Send("Exception occurred: " + e.Message);
                 mDebugger.Send(e.Message);
                 TestController.Failed();
-            }
+            }            
+        }
+
+        public void TestConsoleEncoding()
```

Comment:
the ```Console.ReadKey();``` in this method on line 84, 102 why are they there?

Reply:
This is how I found it, I didnt write that test. I only chnaged "germanic" to "german".

Reply:
I dont think these tests are meant to be run automatic and the ReadKeys allow the developer to check if the output looks correct.